core/os/thread.cpp

```diff
@@ -113,7 +113,9 @@ Thread::Thread() {
 Thread::~Thread() {
 	if (id != _thread_id_hash(std::thread::id())) {
 #ifdef DEBUG_ENABLED
-		WARN_PRINT("A Thread object has been destroyed without wait_to_finish() having been called on it. Please do so to ensure correct cleanup of the thread.");
+		WARN_PRINT(
```

Comment:
I've just reworded this one to be consistent with the warning newly added to `Semaphore`.